backend/src/firebase/firebase_config.ts

```diff
@@ -0,0 +1,11 @@
+import admin, { ServiceAccount } from "firebase-admin";
```

Comment:
We don't need this file. There is already a firebase auth initialization setup in ./util/firebase.ts. Also I've added a new backend env to the folder, essentially I copied the ServiceAccountKey as a variable itself. You will need to use that env file in order for it to work.